config/locales/en.yml

```
@@ -599,6 +599,10 @@ en:
           <p class="govuk-body">You'll need the email or username and password you used to create an account with %{idp_name}.</p>
         continue_verifying: 'Continue verifying with %{idp_name}'
         return_html: 'Or return to <a href="%{rp_start_page}">%{rp_name}</a>.'
+      idp_no_longer_providing_registrations:
+        title: "Verification Paused %{idp_name}"
+        heading: "%{idp_name} are no longer providing registrations"
+        return_html: 'You can return to <a href="%{rp_start_page}">%{rp_name}</a> and register with a different certified company.'
```

```suggestion
        return_html: 'You can try again to <a href="%{rp_start_page}">%{rp_name}</a> and register with a different certified company.'
```
The `name` property in RP config is always an action (verb), see [example](https://github.com/alphagov/verify-hub-federation-config/blob/3858195a3da3f8ef59ef6e41bb7e9e8dd1e018b3/configuration/display-locales/transactions/dvla-view-driving-record.yml#L5)

This also needs @sabrinaharris 's review

---

As above